gobblin-data-management/src/main/java/gobblin/data/management/copy/hive/HiveDataset.java

```diff
@@ -70,47 +63,18 @@
 @Alpha
 public class HiveDataset implements CopyableDataset {
 
-  /**
-   * Specifies a root path for the data in a table. All files containing table data will be placed under this directory.
-   * <p>
-   *   Does some token replacement in the input path. For example, if the table myTable is in DB myDatabase:
-   *   /data/$DB/$TABLE -> /data/myDatabase/myTable.
-   *   /data/$TABLE -> /data/myTable
-   *   /data -> /data/myTable
-   * </p>
-   *
-   * See javadoc for {@link #getCopyableFiles} for further explanation.
-   */
-  public static final String COPY_TARGET_TABLE_ROOT =
-      HiveDatasetFinder.HIVE_DATASET_PREFIX + ".copy.target.table.root";
-  /**
-   * Specifies that, on copy, data files for this table should all be relocated to a single directory per partition.
-   * See javadoc for {@link #getCopyableFiles} for further explanation.
-   */
-  public static final String RELOCATE_DATA_FILES_KEY =
-      HiveDatasetFinder.HIVE_DATASET_PREFIX + ".copy.relocate.data.files";
-  public static final String DEFAULT_RELOCATE_DATA_FILES = Boolean.toString(false);
-
-  public static final String TARGET_METASTORE_URI_KEY = HiveDatasetFinder.HIVE_DATASET_PREFIX + ".copy.target.metastore.uri";
-  public static final String TARGET_DATABASE_KEY = HiveDatasetFinder.HIVE_DATASET_PREFIX + ".copy.target.database";
-
-  private static final Gson gson = new Gson();
-  private static final String databaseToken = "$DB";
-  private static final String tableToken = "$TABLE";
-
-  private final Properties properties;
-  private final FileSystem fs;
-  private final HiveMetastoreClientPool clientPool;
-  private final Table table;
-  private final Splitter splitter = Splitter.on(",").omitEmptyStrings();
+  protected final Properties properties;
+  protected final FileSystem fs;
+  protected final HiveMetastoreClientPool clientPool;
```

Comment:
Needs to be here because it is supplied by the dataset finder, and the `HiveDataset` passes it to the helper at a later time.